.eslintignore

```diff
@@ -11,6 +11,5 @@ tools/test-builds/*
 tests/assets/test-builds/**/*.js
```

Comment:
There's a `post-installer.js` in here

Reply:
Not sure what you mean or are expecting here.